.bmad/bmb/workflows/create-agent/agent-architecture.md

```
@@ -0,0 +1,419 @@
+- `{communication_language}` - Language preference
+- `{date}` - Current date
+- `{module}` - Current module code
+
+### Config Variables
+
+Format: `{config_source}:variable_name`
+Example: `{config_source}:output_folder`
+
+### Path Construction
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)